create-cloud-resources.yml

```diff
@@ -35,6 +35,9 @@
         - username: rattboi
           email: bradon.kanyid@ibm.com
           isadmin: False
+        - username: olaph
```

Comment:
Please move this section up to above rattboi so we can keep our alphabetical ordering.